src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -177,16 +177,21 @@ function SidebarLinks({onLinkClick, insets, optionListItems, isLoading, priority
                 </Tooltip>
                 <SignInOrAvatarWithOptionalStatus isCreateMenuOpen={isCreateMenuOpen} />
             </View>
-
-            <LHNOptionsList
-                style={[isLoading ? styles.flexShrink1 : styles.flex1]}
-                contentContainerStyles={[styles.sidebarListContainer, {paddingBottom: StyleUtils.getSafeAreaMargins(insets).marginBottom}]}
-                data={optionListItems}
-                onSelectRow={showReportPage}
-                shouldDisableFocusOptions={isSmallScreenWidth}
-                optionMode={viewMode}
-            />
-            {isLoading && <OptionsListSkeletonView shouldAnimate />}
+            <View style={[styles.pRelative, styles.flex1]}>
+                <LHNOptionsList
+                    style={styles.flex1}
+                    contentContainerStyles={StyleSheet.flatten([styles.sidebarListContainer, {paddingBottom: StyleUtils.getSafeAreaMargins(insets).marginBottom}])}
+                    data={optionListItems}
+                    onSelectRow={showReportPage}
+                    shouldDisableFocusOptions={isSmallScreenWidth}
+                    optionMode={viewMode}
+                />
+                {isLoading && optionListItems.length === 0 && (
```

Comment:
I see that `optionListItems.length === 0 &&` is fixing skeleton issue.
But were you able to find the root cause why this didn't happen on simulator? More importantly, not happened on other platforms? And more strange thing is that no issue on FlatList.

Reply:
I went through this problem together with @TMisiukiewicz and the reasoning makes sense to me. Flatlist behaved differently and 0px resulted with a 'ok UI state', whereas Flashlist just uncovered it as a bug.